From 6b50b56258ce4c4204a2dffc1aa2405bb59704bd Mon Sep 17 00:00:00 2001
From: Zhang Qiao <zhangqiao22@huawei.com>
Date: Thu, 3 Mar 2022 20:35:06 +0800
Subject: [PATCH] sched/qos: Remove dependency CONFIG_x86
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6b50b56258ce4c4204a2dffc1aa2405bb59704bd
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I4VZJT
CVE: NA

--------------------------------

After removing dependency CONFIG_x86, if enable CONFIG_QOS_SCHED,
only x86 server can handle priority inversion issue.

Signed-off-by: Zhang Qiao <zhangqiao22@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 init/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index 4410b711f9dc..17533f1f19d4 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -953,7 +953,6 @@ config QOS_SCHED
     bool "Qos task scheduling"
     depends on CGROUP_SCHED
     depends on CFS_BANDWIDTH
-    depends on X86
 
     default n
 
-- 
2.26.2

